src/components/Swapping.tsx

```diff
@@ -403,6 +391,23 @@ const Swapping = ({ route, updateRoute, onSwapDone }: SwappingProps) => {
     resumeExecution()
   }
 
+  const switchChainHook = async (requiredChainId: number) => {
+    if (!web3.account || !web3.library) return
+    // check if execution stopped because of a required chain switch
+    const chainSwitchStep = steps.find((step) => step.action.fromChainId === requiredChainId)
```

Comment:
Do we have to switch the chain to claim funds for an nxtp transaction or does that work w/o changing?